The dependencies in bower.json get overwritten by package.json dependencies

When executing `compile` in a project where you have a bower file, the `dependencies` and the `devDevpendencies` get overwritten by the ones specified in package.json

They're overwritten, I wrote a [test case](https://github.com/hvoecking/projectz/commit/d99e3312eb52c06d0eced997d0107d4077685d38), this is the output:
```
FAILURE: 1/2 tests ran successfully; 1 failed, 0 incomplete, 1 errors

Error #1:
projectz-dependencies ➞  files ➞  bower.json
AssertionError: expected '{"dependencies":{"package-dependency-1":"version","package-dependency-2":"version"},"devDependencies":{"package-dev-dependency-1":"version","package-dev-dependency-2":"version"}}' to equal '{"dependencies":{"bower-dependency-1":"version","bower-dependency-2":"version"},"devDependencies":{"bower-dev-dependency-1":"version","bower-dev-dependency-2":"version"}}'
    at /tmp/projectz/out/test/dependencies-test.js:51:59
    at fs.js:271:14
    at Object.oncomplete (fs.js:107:15)

npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0

/tmp/projectz/Cakefile:103
      throw err;
            ^
Error: Process exited with error status code
  at ChildProcess.<anonymous> (/tmp/projectz/Cakefile:92:15)
  at ChildProcess.emit (events.js:98:17)
  at maybeClose (child_process.js:755:16)
  at Process.ChildProcess._handle.onexit (child_process.js:822:5)
```

Are they supposed to get merged?

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,6 +11,7 @@ before_script: "./node_modules/.bin/cake compile"
 script: "npm test"
 node_js:
   - "0.10"
+  - "0.12"
 cache:
   directories:
     - node_modules

--- a/package.json
+++ b/package.json
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "extendr": "~2.1.0",
-    "cson": "~1.6.1",
+    "cson": "~2.0.0",
     "taskgroup": "~4.0.5",
     "getcontributors": "~1.3.4",
     "eachr": "~2.0.2",

--- a/src/lib/projectz.litcoffee
+++ b/src/lib/projectz.litcoffee
@@ -531,8 +531,9 @@ Create the data for the `bower.json` format
 				{
 					name:                   @dataForPackagesMerged.name
 					version:                @dataForPackagesMerged.version
-					dependencies:           @dataForPackagesMerged.dependencies
-					devDependencies:        @dataForPackagesMerged.devDependencies
+					license:                @dataForPackagesMerged.licenses?[0].type
+					description:            @dataForPackagesMerged.description
+					keywords:               @dataForPackagesMerged.keywords
 					main:                   @dataForPackagesMerged.main
 				}
 

--- a/test/out-expected/README.md
+++ b/test/out-expected/README.md
@@ -50,7 +50,7 @@ console.log(result); // 10 - just like normal
 
 // Call the asynchronous function asynchronously
 result = ambi(asyncMethod, 5, 2, function(err,result){ // ambi doesn't do anything special here
-console.log(err, result); // null, 10
+	console.log(err, result); // null, 10
 });
 console.log(result); // setTimeout - just like normal
 ```
@@ -60,14 +60,14 @@ console.log(result); // setTimeout - just like normal
 ### Notes
 
 - Ambi accepts the arguments `(method, args...)`
-- `method` is the function to execute
-- `args...` is the arguments to send to the method
-- the last argument is expected to be the completion callback
-- the completion callback is optional, but if defined, is expected to have the signature of `(err, results...)`
+	- `method` is the function to execute
+	- `args...` is the arguments to send to the method
+		- the last argument is expected to be the completion callback
+		- the completion callback is optional, but if defined, is expected to have the signature of `(err, results...)`
 - If the method has the same amount of arguments as those ambi received, then we assume it is an asynchronous method and let it handle calling of the completion callback itself
 - If the method does not have the same amount of arguments as those ambi received, then we assume it is a synchronous method and we'll call the completion callback ourselves
-- If the synchronous method throws an error or returns an error, we'll try to call the completion callback with a single `err` argument
-- If the synchronous method executes without error, we'll try to call the completion callback with a `err` argument equal to null, and a `result` argument equal to the returned result of the synchronous method
+	- If the synchronous method throws an error or returns an error, we'll try to call the completion callback with a single `err` argument
+	- If the synchronous method executes without error, we'll try to call the completion callback with a `err` argument equal to null, and a `result` argument equal to the returned result of the synchronous method
 - Ambi can also introspect a different method than the one it fires, by passing `[methodToFire, methodToIntrospect]` as the `method` argument
 
 
@@ -78,6 +78,19 @@ console.log(result); // setTimeout - just like normal
 ### [NPM](http://npmjs.org/)
 - Use: `require('ambi')`
 - Install: `npm install --save ambi`
+
+### [Browserify](http://browserify.org/)
+- Use: `require('ambi')`
+- Install: `npm install --save ambi`
+- CDN URL: `//wzrd.in/bundle/ambi@2.1.4`
+
+### [Ender](http://enderjs.com)
+- Use: `require('ambi')`
+- Install: `ender add ambi`
+
+### [Bower](http://bower.io/)
+- Use: `require('ambi')`
+- Install: `bower install ambi`
 
 <!-- /INSTALL -->
 

--- a/test/out-expected/bower.json
+++ b/test/out-expected/bower.json
@@ -1,0 +1,32 @@
+{
+  "name": "ambi",
+  "version": "2.1.4",
+  "authors": [
+    "me"
+  ],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "some-bower-lib": "1.2.3"
+  },
+  "dev-dependencies": {
+    "other-bower-lib": "1.2.3"
+  },
+  "description": "Execute a function ambidextrously (normalizes the differences between synchronous and asynchronous functions). Useful for treating synchronous functions as asynchronous functions (like supporting both synchronous and asynchronous event definitions automatically).",
+  "keywords": [
+    "sync",
+    "async",
+    "fire",
+    "exec",
+    "execute",
+    "ambidextrous",
+    "flow"
+  ],
+  "main": "./out/lib/ambi.js"
+}

--- a/test/src/README.md
+++ b/test/src/README.md
@@ -79,6 +79,19 @@ console.log(result); // setTimeout - just like normal
 - Use: `require('ambi')`
 - Install: `npm install --save ambi`
 
+### [Browserify](http://browserify.org/)
+- Use: `require('ambi')`
+- Install: `npm install --save ambi`
+- CDN URL: `//wzrd.in/bundle/ambi@2.1.4`
+
+### [Ender](http://enderjs.com)
+- Use: `require('ambi')`
+- Install: `ender add ambi`
+
+### [Bower](http://bower.io/)
+- Use: `require('ambi')`
+- Install: `bower install ambi`
+
 <!-- /INSTALL -->
 
 

--- a/test/src/bower.json
+++ b/test/src/bower.json
@@ -1,0 +1,32 @@
+{
+  "name": "ambi",
+  "version": "2.1.4",
+  "authors": [
+    "me"
+  ],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "some-bower-lib": "1.2.3"
+  },
+  "dev-dependencies": {
+    "other-bower-lib": "1.2.3"
+  },
+  "description": "Execute a function ambidextrously (normalizes the differences between synchronous and asynchronous functions). Useful for treating synchronous functions as asynchronous functions (like supporting both synchronous and asynchronous event definitions automatically).",
+  "keywords": [
+    "sync",
+    "async",
+    "fire",
+    "exec",
+    "execute",
+    "ambidextrous",
+    "flow"
+  ],
+  "main": "./out/lib/ambi.js"
+}

